x509/README.md

```diff
@@ -1,6 +1,8 @@
-This package is a port of github.com/golang/go/crypto/x509 package at:
-branch: release-branch.go1.9
-revision: c03ee1985cb6e4467246a2bdb07bb1c62e05f8e9
+Originally based on the go/crypto/x509 standard library,
+this package has now diverged enough that it is no longer
+updated with direct correspondence to new go releases.
 
-Modifications to shared files are located in $name_modified.go to 
-facilitate tracking the stdlib updates. 
\ No newline at end of file
+Approximately supports all the features of
+github.com/golang/go/crypto/x509 package at:
+branch: release-branch.go1.10
+revision: dea961ebd9f871b39b3bdaab32f952037f28cd71
```

Comment:
Do we actually have cipher parity with Golang stdlib, or are we missing the 25519 ciphers (and maybe the new numbers for the Salsa and ChaCha ciphers)?

Reply:
Yes, it looks like we have cipher parity with the Golang stdlib for `/x509`. We cover every signature and public key OID that is covered by them. Interestingly, the golang stdlib `/x509` does not yet support `*25519`, even though it is supported in the stdlib's `/tls`. When #144 lands we will be ahead of them in `*25519` support.

Reply:
We may not be at cipher parity for `/tls` (probably for `*25519`), but we may as well wait until the stdlib supports 1.3 (which could be a while, it's not in the upcoming golang v1.11) before bringing our `/tls` up to date.

Reply:
Good point, I was conflating x509 and tls.